Core/src/Seeding/EstimateTrackParamsFromSeed.cpp

```diff
@@ -30,10 +30,12 @@ Acts::FreeVector Acts::estimateTrackParamsFromSeed(const Vector3& sp0,
   rotation.col(0) = newXAxis;
   rotation.col(1) = newYAxis;
   rotation.col(2) = newZAxis;
+
   // The center of the new frame is at the bottom space point
   Translation3 trans(sp0);
+
   // The transform which constructs the new frame
-  Transform3 transform(trans * rotation);
+  Transform3 transform(trans * Eigen::Isometry3d(rotation));
```

Comment:
This doesn't work without the explicit `Isometry3d`?

Reply:
Without the explicit call to Eigen::Isometry3d(rotation) the result of `trans * rotation` will be interpreted as an affine transformation because a priori it cannot say that the rotation doesn't contain scale and sheer. 
Maybe I could wrap Eigen::Isometry3d(trans*rotation) if you prefer.